src/constants/platform/web.tsx

```diff
@@ -149,8 +185,8 @@ const web = {
           id: 'web-question-4-1',
           content: (
             <>
-              리액트를 사용해서 작은 프로젝트라도 진행해본 경험이 있는 분들을 우선해서 뽑을
-              예정이에요.
+              프로젝트가 기본적으로 리액트를 사용해서 진행되기 때문에, 작은 프로젝트라도 진행해본
+              경험이 있는 분을 우선해서 뽑을 예정입니다.
```

Comment:
```suggestion
              프로젝트가 기본적으로 리액트를 사용해서 진행되기 때문에, 리액트로 작은 프로젝트라도 진행해본
              경험이 있는 분을 우선해서 뽑을 예정입니다.
```
암묵적으로 다 알겠지만 뒤에 어떤것으로 작은 프로젝트를 진행해본 경험인지 연결이 잘 안되는거 같은데 `리액트로`를 붙혀주는건 어떨까유

Reply:
수정 완료했습니다~!